### PR TITLE
Remove make rules to publish to npm.

### PR DESCRIPTION
**Pull request by johnjbarton**
npm run just-publish
does the job now.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -302,53 +302,6 @@ updateSemver: # unless the package.json has been manually edited.
...
-	git fetch upstream
-	-git branch -D upstream_master
-	git checkout -b upstream_master upstream/master
-
-# Now we are on version N with N-1 in npm, update
-
-npm-publish: git-upstream-checkout
-	$(MAKE) clean # sync to the npm version N-1
-	$(MAKE) test  # build version N
-	$(MAKE) dist/commonjs
```
Comment (arv) [inline Makefile]: This is now longer present. Can you add it it package.json?

```json
"rebuild": "make clean && make dist/commonjs && make test",
```

**Comment by arv** [comment]
My goal of publishing a new version was to publish `dist/commonjs`.

File: Makefile (hunk 1 above)
Comment (johnjbarton) [inline Makefile]: done

**Comment by johnjbarton** [comment]
PTAL

**Comment by arv** [comment]
LGTM. Awesome! Looking forward to this getting to npm.

**Pull request closed without merging**